axelrod/strategies/backstabber.py

```diff
@@ -1,7 +1,10 @@
 from axelrod import Player, Actions
 
+from .strategy_transformers import FinalTransformer
+
 C, D = Actions.C, Actions.D
 
+@FinalTransformer([D, D, D]) # End with three defections
```

Comment:
Shouldn't this be two defections? The orginal condition was for opponent history length to be greater than 197, so 198 is the first occurrence. If the the history is 198 long, then we are on turn 199.